metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "title": "Sub-grouping Kho-Bwa",
+    "title": "CLDF dataset derived from Lierberherr and Bodt's \"Comparative Wordlists of Kho-Bwa\" from 2017",
```

Comment:
Really, `Lierberherr`? Can't we pay a bit more attention to the spelling, please?

Reply:
Sorry